src/api/app/controllers/webui/users/rss_token_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Webui
+  module Users
+    class RssTokenController < WebuiController
```

Comment:
The controller should be called `RssTokensController` though to follow [conventions](http://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention).